src/main/resources/assets/admin/common/js/ui/selector/Option.ts

```diff
@@ -1,5 +1,82 @@
-export interface Option<T>
-    extends Slick.SlickData {
+import {IDentifiable} from '../../IDentifiable';
+
+export class Option<T>
+    implements Slick.SlickData, IDentifiable {
```

Comment:
Codacy found an issue: ['Slick' is not defined.](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6422415)